cmd/gateway_controller/addon-manager/addon-manager.go

```diff
@@ -1,9 +1,9 @@
-package main
+package addon_manager
```

Comment:
I'd avoid using underscores in teh package names, see https://go.dev/blog/package-names. It shoudl also match the file name. Personally I'd just call it ocm.

Reply:
renamed to `ocm`